### PR TITLE
Add symptoms repo

### PR DESCRIPTION
**Pull request by i-schuetz**

### CODE REVIEW

File: CoEpi/domain/model/Symptom.swift [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+struct Symptom {
+    let name: String
```
Comment (johnsonh) [inline CoEpi/domain/model/Symptom.swift]: ID?

File: CoEpi/repo/SymptomRepo.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    func symptoms() -> [Symptom]
+
+    func submitSymptoms(symptoms: [Symptom])
+}
+
+class SymptomRepoImpl: SymptomRepo {
+
+    func symptoms() -> [Symptom] {
+        [
+            Symptom(name: "Fever"),
```
Comment (johnsonh) [inline CoEpi/repo/SymptomRepo.swift]: I have the same hardcoded strings, in a follow up we should try to centralize them https://github.com/Co-Epi/app-ios/pull/12/files#diff-0edb6816c821d71dbbd3b88f4d79479cR13

File: CoEpi/repo/SymptomRepo.swift [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+protocol SymptomRepo {
+    func symptoms() -> [Symptom]
```
Comment (johnsonh) [inline CoEpi/repo/SymptomRepo.swift]: getSymptoms?

Comment (johnsonh) [reply CoEpi/repo/SymptomRepo.swift]: Or `retrieveSymptoms`? To parallel with https://github.com/Co-Epi/app-ios/pull/9/files#diff-c91110676bccac94e01ef480f38875ecR7

Comment (i-schuetz) [reply CoEpi/repo/SymptomRepo.swift]: I'm tempted to change `retrieveContacts()` to be only `contacts()`. It's kind of obvious from the context, isn't it?

File: CoEpi/repo/SymptomRepo.swift (hunk 2 above)
Comment (i-schuetz) [reply CoEpi/repo/SymptomRepo.swift]: The repo is currently the central. As a further step the backend could deliver them.

**Comment by i-schuetz** [comment]
I updated the PR.

**Pull request merged**